e2e_tests/tests/cluster/test_rbac.py

```diff
@@ -612,7 +612,7 @@ def test_rbac_describe_role() -> None:
         assert user_assign[1]["roleAssignment"]["scopeWorkspaceId"] == 1
 
 
-@pytest.mark.e2e_cpu
+@pytest.mark.e2e_cpu_rbac
 @pytest.mark.skipif(roles_not_implemented(), reason="ee is required for this test")
```

Comment:
do we still need this skip?

Reply:
Per OOB communication, I'm not really sure. It's nice to support pytesting a directory, but not needed on the CI, and how exactly do we want to support local testing.

Speaking to your (good) recommendation to at least be consistent and skip tests in `test_groups.py`, you're right. That'd be the better choice. Let's pick a policy.

Just... these tests are in kind of a messy state, and it's a surprising amount of work (or a small amount of unnecessary code duplication) to do. I've made the choice to leave the code in this presently-inconsistent state just to get this committed. My hope: it's still a net positive, even if that net isn't as big as it could be.